src/html-purify.js

```diff
@@ -14,7 +14,11 @@ See the accompanying LICENSE file for terms.
         voidElements = whitelist.VoidElements;
 
     function Purifier(config) {
-        var that = this;
+        var that = this,
+            hrefAttributesWhitelist = whitelist.HrefAttributes,
```

Comment:
@maditya please check this list for href attributes, we have 19 entries
https://github.com/yahoo/secure-handlebars/blob/master/src/parser-utils.js